src/registrar/forms/application_wizard.py

```diff
@@ -95,10 +96,39 @@ def to_database(self, obj: DomainApplication):
         """
         raise NotImplementedError
 
+    def test_if_more_than_one_join(self, db_obj, rel, related_name):
```

Comment:
```suggestion
    def has_more_than_one_join(self, db_obj, rel, related_name):
```
Reading this is was a bit difficult to understand when it should return true vs false. (such as when it has multiple joins vs not). Just a rename here would make reading the code intent clear. OR add a "returns True when..." in the comment.
Also, suggest not starting with the word "test" if not a unit test

Reply:
Good suggestion.  Can make this change.

Reply:
done